src/Repository/WishlistRepository.php

```diff
@@ -82,6 +97,19 @@ public function findAllByAnonymous(?string $token): ?array
             ;
     }
 
+    public function countByAnonymous(?string $token): int
```

Comment:
If token is null, for performance improvement, you should not process this request.

Reply:
You're absolutly right. I've also made the change in `findAllByAnonymous`